packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -209,30 +236,59 @@ export default declare((api, options, dirname) => {
           return;
         }
 
-        if (!injectCoreJS2) return;
+        if (!injectCoreJS) return;
 
         if (t.isMemberExpression(parent)) return;
-        if (!has(definitions.builtins, node.name)) return;
-        if (scope.getBindingIdentifier(node.name)) return;
+        if (!hasMapping(definitions.builtins, name)) return;
+        if (scope.getBindingIdentifier(name)) return;
 
-        // Symbol() -> _core.Symbol(); new Promise -> new _core.Promise
+        // transform Symbol(), new Promise
         path.replaceWith(
           this.addDefaultImport(
-            `${modulePath}/core-js/${definitions.builtins[node.name]}`,
-            node.name,
+            `${modulePath}/${corejsRoot}/${definitions.builtins[name].path}`,
+            name,
           ),
         );
       },
 
-      // arr[Symbol.iterator]() -> _core.$for.getIterator(arr)
       CallExpression(path) {
-        if (!injectCoreJS2) return;
+        if (!injectCoreJS) return;
 
-        // we can't compile this
-        if (path.node.arguments.length) return;
+        const { node } = path;
+        const { callee } = node;
 
-        const callee = path.node.callee;
         if (!t.isMemberExpression(callee)) return;
+
+        const { object, property } = callee;
+
+        // transform array.includes()
+        if (!injectCoreJS2 && !hasStaticMapping(object.name, property.name)) {
+          if (hasMapping(definitions.instanceMethods, property.name)) {
+            let context1, context2;
+            if (object.type === "Identifier") {
+              context1 = context2 = object;
```

Comment:
We might need to use `t.cloneNode` here, I'm not sure. You can tell it if some tests are failing complaining about duplicated nodes.